src/aind_data_transfer/jobs/s3_upload_job.py

```diff
@@ -263,6 +265,42 @@ def s3_prefix(self):
             ]
         )
 
+    @staticmethod
+    def _parse_date(date: str) -> str:
+        """Parses date string to %YYYY-%MM-%DD format"""
+        stripped_date = date.strip()
+        pattern = "^\d{4}-\d{2}-\d{2}$"
+        pattern2 = "^\d{1,2}/\d{1,2}/\d{4}$"
+        if re.match(pattern, stripped_date):
+            parsed_date = datetime.strptime(stripped_date, "%Y-%m-%d")
+            return parsed_date.strftime("%Y-%m-%d")
+        elif re.match(pattern2, stripped_date):
+            parsed_date = datetime.strptime(stripped_date, "%m/%d/%Y")
+            return parsed_date.strftime("%Y-%m-%d")
+        else:
+            raise ValueError("Incorrect data format, should be YYYY-MM-DD or DD/MM/YYYY")
+
+    @staticmethod
+    def _parse_time(time: str) -> str:
+        """Parses time string to "%HH-%MM-%SS format"""
+        stripped_time = time.strip()
+        pattern = "^\d{1,2}-\d{1,2}-\d{1,2}$"
+        pattern2 = "^\d{1,2}:\d{1,2}:\d{1,2}$"
+        if re.match(pattern, stripped_time):
+            parsed_time = datetime.strptime(stripped_time, "%H-%M-%S")
+            return parsed_time.strftime("%H-%M-%S")
+        elif re.match(pattern2, stripped_time):
+            parsed_time = datetime.strptime(stripped_time, "%H:%M:%S")
+            return parsed_time.strftime("%H-%M-%S")
+        else:
+            raise ValueError("Incorrect data format, should be HH-MM-SS or HH:MM:SS")
```

Comment:
Should be incorrect `date` instead of `data` format?

Reply:
i had it as data for both date and time, but changing it to date,time accordingly